internal/driver/webui.go

```diff
@@ -63,16 +63,40 @@ func serveWebInterface(port int, p *profile.Profile, o *plugin.Options) error {
 		// only allow requests from local host
 		wrap = checkLocalHost
```

Comment:
If the motivation for this PR is being able to reach the web UI from another host then this check should be changed?

Along these lines, could you update the PR description with the motivation?